cogs5e/models/character.py

```diff
@@ -479,6 +523,34 @@ def get_sheet_embed(self):
                                   f"Please run !update.")
 
         return embed
+    
+    def active_embed(self):
+        """Creates an embed to be displayed when the active character is checked"""
+        embed = EmbedWithCharacter(self)
+        embed.title = self.name
+        urls = {"beyond": "https://ddb.ac/characters/",
+                "dicecloud": "https://dicecloud.com/character/",
+                "google": "https://docs.google.com/spreadsheets/d/"}
+        sheeturl = urls[self.sheet_type] + self.upstream_id
+        desc = f"Your current active character is {active_character.name}. " \
+                "All of your checks, saves and actions will use this character's stats.\n" \
+                "This character is active in " + \
+                (", ".join(["GLOBAL" if self.is_active_global()] +
+                           ["SERVER" if self.is_active_server(ctx)])) + "\n" \
```

Comment:
I'm not a fan of this display - I'd put this info in the embed footer instead in a natural sentence